pom.xml

```diff
@@ -207,11 +207,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>credentials</artifactId>
-            <optional>true</optional>
-        </dependency>
```

Comment:
I guess, that's somehow provided with the BOM?

Reply:
it's a duplicate, declared a bit above